modules/ExchangeApi.py

```diff
@@ -0,0 +1,114 @@
+'''
```

Comment:
For now just the name in the <class=blah> kind of style that you see is fine. We can add more later if we need.